xhtml2pdf/xhtml2pdf_reportlab.py

```diff
@@ -32,10 +32,9 @@
 try:
     import StringIO
 except Exception:
-    from io import StringIO
-    StringIO_old = StringIO
+    from io import BytesIO
     class StringIO(object):
-        StringIO = StringIO_old
+        StringIO = BytesIO
```

Comment:
Agreed, 2.5 and 2.6 are no longer supported by Python. This branch is meant to just get Py3 support with no new features.

After this is in, if people want new features they really should upgrade.